modules/ROOT/pages/intro-engine.adoc

```diff
@@ -9,7 +9,9 @@ From a development point of view, there are a few major changes:
 
 * Exchange patterns no longer exist. All connectors receive responses. If you want to process a message asynchronously, use the Async component.
 * Every flow always uses a non-blocking processing strategy, and there is no need to configure processing strategies anymore.
-* There is a single, global thread pool for all flows.
+* We now have three centralized pools. All three are managed by the Mule runtime and shared across all applications deployed to that runtime.
```

Comment:
while true, these 3 points may be irrelevant to someone developing an app. Better to put a note with a link to `https://docs.mulesoft.com/mule-runtime/4.1/execution-engine` for more details.